extensions/amp-ima-video/amp-ima-video.md

```diff
@@ -66,14 +66,14 @@ VAST-compliant ad response (for examples, see
 
 ## Attributes
 
-**data-src** (required)
-
-The URL of your video content.
-
 **data-tag** (required)
 
 The URL for your VAST ad document.
 
+**data-src** (required if no `<source>` children are present)
+
+DEPRECATED Prefer `<source>` child elements. The URL of your video content.
```

Comment:
I wound't call it deprecated. Just `The URL of your video content.` should be good enough